.github/workflows/int-test-workflow.yml

```diff
@@ -0,0 +1,150 @@
+name: Integration Test WorkFlow
+on:
+  schedule:
+    - cron: "0 12 * * *" 
```

Comment:
nit: Does it make sense to run at the end of the day, instead of at the middle of the day, so that it covers all the merges for that day?